BoneCentral/Brain/app.js

```diff
@@ -0,0 +1,24 @@
+var Config              = require('./config.json');
+var Ports               = require('./Sockets/Ports.json');
+var Sockets             = require('./Sockets');
+var Peripherals        	= require('./CppInterface/PeripheralsProcess.js');
+var HardwareFactory     = require('./CppInterface/Factory.js');
+var Vision              = require('./VisionInterface');
+
+var server              = Sockets.createSocket(Ports.DispatcherPort);
```

Comment:
This is breaking encapsulation. These sockets should have been left inside the Factory.